src/utils/XhrNetworkInterceptor.ts

```diff
@@ -24,11 +24,11 @@ export interface NetworkData {
   gqlQueryName?: string;
   serverErrorMessage: string;
   requestContentType: string;
-  w3cc: boolean | null;
-  partialId: number | null;
-  etst: number | null;
-  wgeti: string | null;
-  wceti: string | null;
+  isW3cHeaderFound: boolean;
+  partialId: number;
+  networkStartTimeInSeconds: number;
```

Comment:
We already collect the start time in `startTime` but in milliseconds, so I think we can just use `network.startTime / 1000` to get the time in seconds instead of storing it in a different field to avoid discrepancy.

Reply:
network start time in seconds are used in many places so its better to call networkStartTimeInSeconds rather than do a calculation every time

Reply:
I feel like a getter would be more suitable here instead of a different attribute since now the values could easily go out of sync.

We could do something like this:

```typescript
interface NetworkData {
  // ...
  startTime: number;
  startTimeInSeconds: number;
}

const network: Network = {
  // ...
  startTime: 12000,
  get startTimeInSeconds() {
    return Math.floor(this.startTime / 1000);
  },
};
```